plugin/neomake.vim

```diff
@@ -21,35 +21,79 @@ command! -bang NeomakeCancelJobs call neomake#CancelJobs(<bang>0)
 
 command! -bang -bar NeomakeInfo call neomake#debug#display_info(<bang>0)
 
-" Enable/disable/toggle commands.  {{{
-function! s:toggle(scope) abort
-    let new = !get(get(a:scope, 'neomake', {}), 'disabled', 0)
-    if new
-        call neomake#config#set_dict(a:scope, 'neomake.disabled', new)
-    else
-        call neomake#config#unset_dict(a:scope, 'neomake.disabled')
+function! s:status_feedback(disabled, where, verbose) abort
+    let msg = (a:disabled ? 'disabled' : 'enabled').' '.a:where.'.'
+    if a:verbose
+        echom 'Neomake: '.msg
     endif
-    call s:display_status()
-    call neomake#statusline#clear_cache()
+    call neomake#log#debug(msg)
 endfunction
```

Comment:
Something like:
```
Neomake: Disabled [Buffer: Disabled] [Tab: Enabled] [Global: Enabled]
```

Ordered by specificity. Show only the parts that are explicitly configured, but always keep the first one, the "effective" status.

Since toggling comes from a keybinding or command, there is no need to hide it under the `verbose` flag.

Reply:
@somini 
Came back to this finally, integrating your suggestion.
What do you think?

Reply:
Looks good to me.

Having two parts configured should be somewhat rare, so I'm OK with keeping the current string when a single part is configured.